src/components/PageTitle.jsx

```diff
@@ -1,7 +1,22 @@
-const PageTitle = () => {
+import Image from "next/image";
+import wave from "@/public/waveBottom.webp";
+
+const PageTitle = ({ image, title }) => {
   return (
-    <div className="">
-      <p>PageTitle</p>
+    <div className="w-full relative">
+      <Image
+        src={image}
+        alt="background image"
+        className="w-screen opacity-80"
+      />
+      <p className="absolute top-[0%] pl-[10%] font-bold text-5xl text-white border-swim-yellow border-r-[20px] pr-[10%] py-[15%] bg-swim-blue-200/70">
```

Comment:
![image](https://github.com/acm-ucr/ucr-swim-website/assets/91030482/514fb49f-c65f-48ff-bb95-0f318b4af816)
try to avoid padding for this instance since it can break for different screen sizes.

- use width and height to change thee size of the blue tint
- you can use flex here. it might seem counterintuitive since we also use absolute here, but flex applies to items inside the div tag. center using justify and items.

Reply:
top-[0%] can be simplified to top-0